### DISCUSSION TITLE
Exit on single <c-[> possible?

### DISCUSSION THREAD

**Discussion by znxuz** [mcchrish/nnn.vim#118] [Q&A] [answered]
I've tried mapping <c-[> to q/close/quit but they all just wouldn't work. It is even possible to just hit single <c-[>to exit? I know the default behaviour of both nnn and this plugin is to hit '[' two times but muscle memory...

**Comment by mcchrish** [comment, marked as answer]
You can define mapping to nnn window with autocmd. Please try:
```vim
autocmd FileType nnn tnoremap <nowait><buffer><silent> <C-]> q
```

**Reply by znxuz** [reply]
Works flawlessly, thanks!
